src/RecyclableMemoryStream.cs

```diff
@@ -999,17 +993,17 @@ public override void Write(byte[] buffer, int offset, int count)
             if (offset < 0)
             {
                 throw new ArgumentOutOfRangeException(nameof(offset), offset,
-                                                      "Offset must be in the range of 0 - buffer.Length-1");
+                                                      $"{nameof(offset)} must be in the range of 0 - {nameof(buffer)}.Length-1.");
```

Comment:
Should we give `.Length` the `nameof()` treatment as well?

Reply:
That will guarantee that docs won't be out of sync with whatever is being used at the moment.